.github/workflows/publish.yml

```diff
@@ -0,0 +1,43 @@
+name: Publish Gradle plugin
+
+on: workflow_dispatch # only run manually
+
+env:
+  PANTS_SHA: "33735fe23228472367dc73f26bb96a755452192f"
+  IJ_ULTIMATE: false
+  CI: true
+
+jobs:
+  build:
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v1
+      - name: Prepare
+        run: sudo apt-get install -y openjdk-8-jdk unzip curl python3-dev python3-virtualenv pkg-config git g++ libssl-dev make g++ zip
+      - uses: actions/setup-java@v2
+        with:
+          distribution: 'zulu'
+          java-version: '11'
+      - name: Cache .cache
+        uses: actions/cache@v2
+        with:
+          path: .cache
+          key: pants-${{ env.PANTS_SHA }}
+      - name: Setup CI
+        run: PANTS_SHA=${{ env.PANTS_SHA }} IJ_ULTIMATE=${{ env.IJ_ULTIMATE }} ./scripts/setup-ci-environment.sh
+      - name: Test
+        run: |
+          export CWD=$(pwd)
```

Comment:
minor (worth doing if there is something more to fix). Extract this to something like ./scripts/ci_run_gradle.sh to avoid repeating this exports

Reply:
We should migrate all those scripts to Gradle and give them reasonable defaults, this way we can drop those exports